src/hooks/useBridgedBlocks.ts

```diff
@@ -37,8 +37,7 @@ interface Props {
 const useBridgedBlocks = ({ isApiReady, api, chain }: Props) => {
   const [importedHeaders, setImportedHeaders] = useState('');
 
-  //const bridgedChain = `bridge${chain}`;
-  const bridgedChain = `bridge${chain}`;
+  const bridgedChain = `bridge${chain}Grandpa`;
```

Comment:
I'd if all chain-related names would be grouped in a single file (like constants). This may change in the future, and I'd rather not look for string concats in the entire codebase.
```js
function getGrandpaPalletName(chain) {
  return `bridge${chain}Grandpa`;
}
```

Reply:
i know what you mean and i agree... but this concatenation is used only once across the application and only in this file.
I feel that now is simpler having the constant in the file and going to this file specifically and change the name in case it is needed. But of course that i'm opened to implement that function if it worths it.